templates/terraform/resource.erb

```diff
@@ -245,6 +249,7 @@ func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{
 <%= lines(compile(object.custom_code.post_create)) if object.custom_code.post_create -%>
 
     return resource<%= resource_name -%>Read(d, meta)
+<%  end # if custom_create  -%>
```

Comment:
I assume you are going to straight call the Update method that returns Read anyway?